dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -102,29 +102,28 @@ public static void main(String[] arguments)
     // TODO(suztomo): print the report
     StaticLinkageCheckReport report = staticLinkageChecker.findLinkageErrors();
 
-    printStaticLinkageErrors(unresolvedMethodReferences);
+    printStaticLinkageReport(report);
   }
 
-  private static void printStaticLinkageErrors(
-      List<FullyQualifiedMethodSignature> unresolvedMethodReferences) {
-    if (unresolvedMethodReferences.isEmpty()) {
-      System.out.println("There were no unresolved method references");
-      return;
-    }
-    ImmutableSortedSet<FullyQualifiedMethodSignature> sortedUnresolvedMethodReferences =
-        ImmutableSortedSet.copyOf(unresolvedMethodReferences);
-    int count = sortedUnresolvedMethodReferences.size();
-    Formatter formatter = new Formatter();
-    formatter.format(
-        "There were %,d unresolved method references from the jar file(s):\n", count);
-    for (FullyQualifiedMethodSignature methodReference : sortedUnresolvedMethodReferences) {
-      formatter.format(
-          "Class: '%s', method: '%s' with descriptor %s\n",
-          methodReference.getClassName(),
-          methodReference.getMethodSignature().getMethodName(),
-          methodReference.getMethodSignature().getDescriptor());
+  private static void printStaticLinkageReport(StaticLinkageCheckReport report) {
```

Comment:
This looks like it could become a method on `StaticLinkageCheckReport` instead. And maybe the stuff in the loop could become a method on `JarLinkageReport`.

Reply:
I prefer to keep these print-related code in one place. It can easily controlling indent in this way.
Is this ok with you?

Reply:
Sure.